plugins/entando-plugin-jpkiebpm/src/main/java/com/agiletec/aps/system/services/page/KiePageManager.java

```diff
@@ -48,9 +48,9 @@ public void removeWidget(String pageCode, int pos) throws ApsSystemException {
         try {
             Widget widget = this.getDraftPage(pageCode).getWidgets()[pos];
             super.removeWidget(pageCode, pos);
-            String stringId = widget.getConfig().getProperty(KieBpmSystemConstants.WIDGET_PARAM_DATA_UX_ID);
+            String stringId = widget != null && widget.getConfig() != null ? widget.getConfig().getProperty(KieBpmSystemConstants.WIDGET_PARAM_DATA_UX_ID) : null;

```

Comment:
See prior comment on the same(?) PR. Change this to an if statement with a setter. Tough to read multi line trinary operators.